doc/decisions/0003-geth-node.md

```diff
@@ -0,0 +1,83 @@
+# 0003. Geth node for Beta
+
+| Date | Tags |
+|---|---|
+| 2018-05-14 | architecture, p2p, scope   |
+
+
+## Status
+
+proposed
+
+## Context
+
+On the path to releasing a usable beta we've faced a variety of performance
+issues (https://github.com/status-im/ideas/issues/55,
```

Comment:
Just nitpicking, can you create a list for better readability?

Reply:
done